src/par_iter/slice_mut.rs

```diff
@@ -0,0 +1,67 @@
+use super::*;
+use super::len::ParallelLen;
+use super::state::ParallelIteratorState;
+use std::mem;
+
+pub struct SliceIterMut<'r, T: 'r + Send> {
```

Comment:
What method are you using to name these lifetimes? I thought `'a` was the standard convention?